lib/mandrill_dm/message.rb

```diff
@@ -65,6 +65,10 @@ def merge_language
       return_string_value(:merge_language)
     end
 
+    def merge_vars
+      mail[:merge_vars] ? mail[:merge_vars].instance_variable_get('@value') : nil
```

Comment:
why do you need to use `instance_variable_get('@value')`?  Does `mail[:merge_vars].value` not have what you need?